test/new-e2e/tests/npm/common_1host.go

```diff
@@ -24,6 +25,33 @@ func test1HostFakeIntakeNPM[Env any](v *e2e.BaseSuite[Env], FakeIntake *componen
 	t := v.T()
 	t.Helper()
 
+	t.Cleanup(func() {
```

Comment:
Is this something we can add the the suite cleanup? It seems broadly useful for debugging failed tests.

Reply:
I can run this in the `suite.AfterTest()` but this would help only debug `test1HostFakeIntakeNPM` a the moment